### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by adam50200** [pkgxdev/pkgx#922] [Help]
I have made a translation into Azerbaijani
https://github.com/pkgxdev/ossapp/pull/832#issuecomment-1883473376

but it was not added here

https://github.com/pkgxdev/ossapp/tree/main/svelte/src/libs/translations/languages

Maybe I did something wrong. Tell me

**Comment by adam50200** [comment]
https://github.com/pkgxdev/ossapp/pull/843

**Comment by dbentry** [comment]
Are more translations needed?

**Reply by adam50200** [reply]
Your-no

**Comment by berllx** [comment]
Is there a way to participate in the translation of the documentation/website?

**Comment by berllx** [comment]
Nvm. Thanks for idea

**Reply by adam50200** [reply]
You don’t

**Comment by adam50200** [comment]
@mxcl 

close this topic too, you are bad people, you don't even want to help
